src/services/configServices/SettingsService.js

```diff
@@ -63,7 +63,7 @@ class SettingsService {
         mergedConfigContent.url !== "" &&
         !mergedConfigContent.url.startsWith("https://")
       ) {
-        mergedConfigContent.url = "https://" + mergedConfigContent.url
+        mergedConfigContent.url = `https://${mergedConfigContent.url}`
```

Comment:
Just to clarify, this change was done for your personal preference in code right? Correct me if I am wrong but `"https://" + mergedConfigContent.url`  and  ``https://${mergedConfigContent.url}`` are semantically equivalent right?

Reply:
Yup, this was actually auto modified by our linter I think